src/sentry/api/endpoints/group_events.py

```diff
@@ -104,9 +104,14 @@ def _get_events_snuba(self, request, group, environments, query, tags, start, en
         )
 
         serializer = EventSerializer() if full else SimpleEventSerializer()
+
+        def handle_results(results):
+            eventstore.bind_nodes(results)
```

Comment:
Yes, we have moved the multi get bind_nodes out of the serializer into the only 3 endpoints where it would ever be useful. There are no other places where we call bind_nodes on an event now.